mix.exs

```diff
@@ -35,6 +35,7 @@ defmodule OMG.Umbrella.MixProject do
         git: "https://github.com/omisego/ex_unit_fixtures.git", branch: "feature/require_files_not_load", only: [:test]
       },
       {:ex_doc, "~> 0.20.2", only: :dev, runtime: false},
+      {:statix, "~> 1.1"},
       {:appsignal, "~> 1.9"},
```

Comment:
Should we be removing `appsignal` here too?

Reply:
until we remove the appsignal from the decorator, it still has to stay